mpi-proxy-split/p2p_drain_send_recv.cpp

```diff
@@ -228,8 +239,8 @@ drainSendRecv()
   while (!allDrained()) {
     sleep(1);
     int numReceived = 0;
-    // If pending MPI_Irecvs, use MPI_Test in case msg was sent.
-    numReceived += completePendingIrecvs();
+    // If pending MPI_Irecvs or MPI_Isend, use MPI_Test to try to complete them.
```

Comment:
`pending MPI_Irecvs` -> `pending MPI_Irecv`
(I know this is a previous comment.  But let's polish it further now.)

`to complete them` -> `to complete it`  ("A or B" is singular, and "A and B" is plural.)